modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/SubscriptionDLAppHelperLocalServiceWrapper.java

```diff
@@ -115,6 +119,22 @@ public void updateStatus(
 		}
 	}
 
+	@Activate
+	protected void activate() {
+		_closeable = _liferayJSONDeserializationWhitelist.register(
+			DLSubscriptionSender.class.getName());
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		try {
+			_closeable.close();
+		}
+		catch (Exception exception) {
+			throw new RuntimeException(exception);
+		}
+	}
+
```

Comment:
Why are you wrapping the exn in a runtime exn? If this is non intentional, you can simply declare the exception in the method signature.

Reply:
Also, there's a NPE here (see my next comment).

Reply:
@adolfopa, I updated. Please help me take a look. Thanks!